packages/scandipwa/src/query/Config.query.js

```diff
@@ -40,6 +41,13 @@ export class ConfigQuery {
             ]);
     }
 
+    getCurrencyRates() {
+        return new Field('currency')
+            .addField('base_currency_code')
+            .addField(new Field('exchange_rates')
+                .addFieldList(['currency_to', 'rate']));
```

Comment:
Please use the `addFieldList` to add multiple fields and move the field list to the separate methods for better extensibility.